lib/active_model/serializer.rb

```diff
@@ -105,7 +105,7 @@ def self.root_name
       name.demodulize.underscore.sub(/_serializer$/, '') if name
     end
 
-    attr_accessor :object, :root, :meta, :meta_key, :scope
+    attr_accessor :object, :root, :meta, :meta_key, :links, :scope
```

Comment:
:100: 

There is a method to [add_links in #1041](https://github.com/rails-api/active_model_serializers/pull/1041/files#diff-ae1e38e96a8bf28ae37918a65c448c73R164) , so a think we can use it, right?